### ISSUE TITLE
TBG Feature request #634 (hishutup on 2014-05-13): Search function

### ISSUE THREAD

**Issue by TanninOne** [TanninOne/modorganizer#107]
It would be great to have a universal search function with categories, such as, downloads, installed mods, uninstalled mods, overwrite. Include sub categories for example, searching for a file name 'in' installed list 'in' textures folder.  Even a simple search would be nice.

**Comment by user21680** [comment]
I agree. I vouch for a Search function to search in all the fields (name, category, id, notes, etc), but with the option to search in a specific field, e.g., "category:weapons" or "c:weapons", which would return only the mods of that category. The filters already allow this, but they don't allow more complex filtering like: "-c:weapons", "c:weapons content:textures -content:meshes", "notes:MCM".
